.github/workflows/build.yml

```diff
@@ -34,6 +34,10 @@ jobs:
     - name: Test
       run: |
         pytest
+    - name: Test with lxml
+      run: |
+        pip install lxml
```

Comment:
Just FYI, I've noticed that the `publish:` job seems to install lxml:

`pip install lxml flake8 pytest coverage`

Reply:
That's surprising, wouldn't have expected it there. I think this makes it more obvious.

Reply:
You have left this line in the new `coverage-lint` job:

`pip install lxml flake8 pytest coverage`

which means that coverage will be tested using `lxml`. I guess that is fine, but just pointing it out.